stl/inc/ranges

```diff
@@ -3042,10 +3042,11 @@ namespace ranges {
         /* [[no_unique_address]] */ _Non_propagating_cache<_Cache_wrapper, false> _Inner{};
     };
 
-    // clang-format off
+

```

Comment:
Not worth resetting testing, but if we need to change anything else we could also:
```suggestion
```

Reply:
Fixed.